include/cglm/struct/mat4.h

```diff
@@ -459,4 +459,17 @@ glms_mat4_(rmc)(vec4s r, mat4s m, vec4s c) {
   return glm_mat4_rmc(r.raw, m.raw, c.raw);
 }
 
+/*!
+ * @brief Given a 16 element float array create mat4 matrix
+ *
+ * @param[in]  src  pointer to float array
+ * @param[out] dest matrix
+ */
+CGLM_INLINE
+mat4s
+glm_mat4_(make)(float * __restrict src, mat4s dest) {
```

Comment:
You should use `glms_` namespace/prefix here not `glm_` which indicates inline array api.

Reply:
Thanks changes made